h/views/api/helpers/links.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""Helpers for generating and retrieving service link metadata"""
+
+from __future__ import unicode_literals
+
+
+class ServiceLink(object):
+    """Encapsulate metadata about an API service"""
+
+    def __init__(self, name, route_name, method="GET", description=None):
+        self.name = name
+        self.route_name = route_name
+        self.method = method
+        self.description = description
+
+    def primary_method(self):
```

Comment:
I left this as a public method because it seemed legit that an external party might be interested in a link's method. Plus I am a proponent of public-unless-good-reason-not-to.